mappings/net/minecraft/client/gui/CommandBlockScreen.mapping

```diff
@@ -1,7 +1,12 @@
 CLASS cvx net/minecraft/client/gui/CommandBlockScreen
-	FIELD A typeButton Lcrm;
+	FIELD A modeButton Lcrm;
 	FIELD B conditionalModeButton Lcrm;
 	FIELD C redstoneTriggerButton Lcrm;
-	FIELD D type Lbpi$a;
+	FIELD D mode Lbpi$a;
 	FIELD E conditional Z
 	FIELD F alwaysActive Z
+	FIELD z blockEntity Lbpi;
+	METHOD v updateMode ()V
+	METHOD w cycleType ()V
+	METHOD x updateConditionalMode ()V
+	METHOD y updateAlwaysActive ()V
```

Comment:
Will `updateRedstoneMode` be better?

Reply:
I think so, but would like a 2nd opinion

Reply:
Not sure. Maybe `updateActivationMode`?

Reply:
No, a 2nd opinion - as in an opinion from a person who is neither me nor you

Reply:
Ahh

Reply:
`updateActivationMode` seems good

Reply:
I agree `updateActivationMode` does seem good

Reply:
Done.